modules/light-clients/06-solomachine/client_state_test.go

```diff
@@ -541,6 +541,13 @@ func (suite *SoloMachineTestSuite) TestVerifyMembership() {
 				},
 				false,
 			},
+			{
+				"empty path",
+				func() {
+					path = []byte("")
+				},
+				false,
```

Comment:
I think this depends on https://github.com/cosmos/ibc-go/pull/2736 as `path` will change from `[]byte` to `exported.Path`

Reply:
#2736 is now merged, you may be able to change this to the below to cover the case:

```go
path = commitmenttypes.MerklePath{}
```